File: Integer to Roman/solve.js
/**
 * @param {number} num
 * @return {string}
 */
var intToRoman = function (num) {
    let ans = "";
    const symbol = [
        "M",
        "CM",
        "D",
        "CD",
        "C",
        "XC",
        "L",
        "XL",
        "X",
        "IX",
        "V",
        "IV",
        "I",
    ];
    const value = [1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1];
    for (let i = 0; i < 13; i++) {
        while (value[i] <= num) {
            num -= value[i];
            ans += symbol[i];
        }
    }
    return ans;
};

File: Integer to Roman/solve.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution
{
public:
    string intToRoman(int num)
    {
        string ans;
        string symbol[] = {"M", "CM", "D", "CD", "C", "XC", "L", "XL", "X", "IX", "V", "IV", "I"};
        int value[] = {1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1};
        for (int i = 0; i < 13; i++)
        {
            while (value[i] <= num)
            {
                num -= value[i];
                ans += symbol[i];
            }
        }
        return ans;
    }
};